### DISCUSSION TITLE
Precisamos escrever o roteiro do jogo e decidir a historia.

### DISCUSSION THREAD

**Discussion by CodyKoInABox** [Equipe-CyberPAC/discussions#1] [Ideas]
O titulo e auto explicativo. Precisamos decidir como sera a historia do jogo e escrever um roteiro base para iniciarmos o desenvolvimento.

**Comment by JessicaRoecker** [comment]
Que tal começar assim:  

No ano de 2135, o protagonista, John Turner, um cientista talentoso, está testando uma nova invenção revolucionária: uma máquina do tempo. Durante o teste inicial, algo dá errado e a máquina do tempo falha, deixando John preso em um ciclo temporal que o leva a diferentes momentos da história da Terra. Além disso, a máquina do tempo está danificada e precisa ser consertada com a ajuda de cinco joias especiais que estão espalhadas ao longo do tempo.
